tutorials/terraform/data-transfer/postgresql-to-clickhouse.tf

```diff
@@ -0,0 +1,260 @@
+# Infrastructure for Yandex Cloud Managed Service for ClickHouse cluster, Yandex Cloud Managed Service for PostgreSQL, and Data Transfer
+#
+# RU: https://cloud.yandex.ru/docs/data-transfer/tutorials/rdbms-to-clickhouse
+# EN: https://cloud.yandex.com/en/docs/data-transfer/tutorials/rdbms-to-clickhouse
+
+# Specify the following settings
+locals {
+  ch_password = "" # Set a password for the ClickHouse admin user
+  pg_password = "" # Set a password for the PostgreSQL admin user
+
+  transfer_enabled = 0 # Set to 1 ONLY AFTER a table in source cluster is created. Then run "terraform apply" command again to enable the transfer
```

Comment:
Set to 1 ONLY AFTER a table in the source cluster is created. Then run the "terraform apply" command again to enable the transfer